.controlplane/Dockerfile

```diff
@@ -17,18 +17,20 @@ RUN bundle config set without 'development test' && \
     bundle install --jobs=3 --retry=3
 
 # install node packages
-COPY package.json yarn.lock .
+COPY package.json yarn.lock ./
 RUN yarn install
 
 # pick necessary app files
-COPY Gemfile* config.ru Rakefile babel.config.js ./
-COPY app ./app
-COPY bin ./bin
-COPY client ./client
-COPY config ./config
-COPY db ./db
-COPY lib ./lib
-COPY public ./public
+# COPY Gemfile* config.ru Rakefile babel.config.js postcss.config.js ./
+# COPY app ./app
+# COPY bin ./bin
+# COPY client ./client
+# COPY config ./config
+# COPY db ./db
+# COPY lib ./lib
+# COPY public ./public
+
+COPY . ./
```

Comment:
Either we should copy all files (similar to the old Docker file at the root of the project) or we should mention all the config files in the root directory.

I intentionally kept the old lines commented to decide about it.

Reply:
do this in a followup

Reply:
if cannot be done within 12 hours

Reply:
Done